test/mortgage_e2e.ipynb

```diff
@@ -0,0 +1,1123 @@
+    "col_names_path = data_dir + \"names.csv\""
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/home/taurean/miniconda3/envs/rapids013-323-test/lib/python3.6/site-packages/fsspec/implementations/local.py:33: FutureWarning: The default value of auto_mkdir=True has been deprecated and will be changed to auto_mkdir=False by default in a future release.\n",
+      "  FutureWarning,\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "                                   seller_name\"|\"new\n",
+      "0                        WITMER FUNDING|LLC\"|\"Witmer\n",
+      "1  WELLS FARGO CREDIT RISK TRANSFER SECURITIES TR...\n",
+      "2                 WELLS FARGO BANK| NA\"|\"Wells Fargo\n",
+      "3                WELLS FARGO BANK|N.A.\"|\"Wells Fargo\n",
+      "4                  WELLS FARGO BANK|NA\"|\"Wells Fargo\n"
+     ]
+    }
+   ],
+   "source": [
+    "#test to see you are reading from the proper directory.  You will see an output if you are.\n",
+    "temp = cudf.read_csv(col_names_path)\n",
+    "print(temp.head())\n",
+    "del temp"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def initialize_rmm_pool():\n",
+    "    import rmm\n",
+    "\n",
+    "    rmm.reinitialize(pool_allocator=True)\n",
+    "\n",
+    "def initialize_rmm_no_pool():\n",
+    "    import rmm\n",
+    "\n",
+    "    rmm.reinitialize(pool_allocator=False)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'tcp://127.0.0.1:33433': None, 'tcp://127.0.0.1:40907': None}"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "client.run(initialize_rmm_pool)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "#### Define functions to encapsulate the workflow into a single call"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 10,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "#You can edit your number of dask_cudf partitions here, depending on how many GPUs you have and how big your dataset is.  The number chosen can boost or reduce performance\n",
+    "n_partitions = 10"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 11,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def run_dask_task(func, **kwargs):\n",
+    "    task = func(**kwargs)\n",
+    "    return task\n",
+    "\n",
+    "def process_quarter_gpu(year=2000, quarter=1, perf_file=\"\"):\n",
+    "    ml_arrays = run_gpu_workflow(quarter=quarter,year=year, perf_file=perf_file)\n",
+    "    return ml_arrays\n",
```

Comment:
Can probably get rid of this and call `run_gpu_workflow` directly from the caller

Reply:
- [ ] removed